coalib/bearlib/languages/documentation/DocumentationExtraction.py

```diff
@@ -1,3 +1,12 @@
+"""Functions that extract doc comments.
+
+The only public function extract_documentation_with_markers() is used by
+DocBaseClass.extract(). All it does is going over the code line by line
+and calling _extract_doc_comment_from_line(), which calls
+_extract_doc_comment(), which delegates extracting docs to three more
+specific functions: _extract_doc_comment_simple(),
+_extract_doc_comment_continuous() and _extract_doc_comment_standard().
+"""
```

Comment:
Just an idea, you can change it however you feel like. Also, make sure you adhere to 79 char line limit though